pages/_document.js

```diff
@@ -58,33 +59,25 @@ export default class Page extends Document {
               content="width=device-width, initial-scale=1.0, maximum-scale=1.0,
               user-scalable=0"
             />
-            <link
-              rel="shortcut icon"
-              type="image/x-icon"
-              href="/static/favicon.ico"
-            />
+            <link rel="shortcut icon" type="image/x-icon" href="/favicon.ico" />
             <link
               rel="shortcut icon"
               type="image/vnd.microsoft.icon"
-              href="/static/favicon.ico"
+              href="/favicon.ico"
             />
             <link
               rel="icon"
               type="image/png"
-              href="/static/favicon-32x32.png"
+              href="/favicon-32x32.png"
               sizes="32x32"
             />
             <link
               rel="icon"
               type="image/png"
-              href="/static/favicon-16x16.png"
+              href="/favicon-16x16.png"
               sizes="16x16"
             />
-            <link
-              rel="stylesheet"
-              type="text/css"
-              href="/static/fonts/fonts.css"
-            />
+            <link rel="stylesheet" type="text/css" href="/fonts/fonts.css" />
```

Comment:
same here - it's all part of `static` probably

Reply:
Fixed